server/services/netatmo/lib/event/netatmo.pollManual.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @description Poll value of a Netatmo devices
+ * @example
+ * pollManual();
+ */
+async function pollManual() {
```

Comment:
I don't understand what this function does

Every time pollManual is called, it adds new interval? it means there will be tons of interval created, weird!

Reply:
Oh yes I see ... It's true if we spam "connect" but I see there is an issue.